src/app/layout/_variable.scss

```diff
@@ -0,0 +1 @@
+$fontFamily: Arial, Helvetica, sans-serif;
```

Comment:
I would suggest to shift colors variables from _colors to here because this is the file which will contains all the variables. just make sure to separate the variables with proper comments. For example /*Variables for color*/, /*Variables for font-size*/ etc